docs/iris/src/userguide/interpolation_and_regridding.rst

```diff
@@ -150,81 +150,81 @@ values requiring extrapolation being masked.
 
 .. _regridding:
 
-Regridding
----------------------------------
 
-Regridding conceptually is a very similar to interpolation in Iris, with the primary difference being
-that interpolations are based on sample points, where regridding is based on the **spatial** grid of
-*another cube*.
+Regridding
+----------
 
-Regridding is achieved with the :meth:`cube.regrid() <iris.cube.Cube.regrid>` method,
-with the first argument being *another cube* which has the grid to which the cube should
-be interpolated onto, and the second argument being the regridding scheme to use.
+Regridding is conceptually a very similar process to interpolation in Iris. 
+The primary difference is that interpolation is based on sample points, while
+regridding is based on the **spatial** grid of *another cube*.
```

Comment:
Is it worth while stressing that it's `horizontal grid` ... or would we be in danger of preaching to the converted?